File: scripts/py-compile-suite/inputs/24_sort_unique.m
% Sort and unique
x = [5, 3, 8, 1, 9, 3, 5, 2, 8];
sorted_asc = sort(x);
sorted_desc = sort(x, 'descend');
[sorted_vals, idx] = sort(x);
unique_vals = unique(x);
n_unique = length(unique_vals);

File: scripts/py-compile-suite/outputs/24_sort_unique.py
import numpy as np
from matlabtopython_compat import sort_with_index

# Sort and unique
x = np.array([5, 3, 8, 1, 9, 3, 5, 2, 8])
sorted_asc = np.sort(x)
sorted_desc = np.sort(x, 'descend')
sorted_vals, idx = sort_with_index(x)
unique_vals = np.unique(x)
n_unique = np.max(unique_vals.shape)
